docs/cel2/notices/pectra-upgrade.md

```diff
@@ -1,13 +1,13 @@
 # Preparing for Pectra L1 hardfork
 
-This page outlines breaking changes related to the Ethereum Pectra (Prague-Electra) L1 hard fork for node operators on the Celo Alfajores L2 testnet.
+This page outlines breaking changes related to the Ethereum Pectra (Prague-Electra) L1 hard fork for node operators on the Celo Alfajores & Baklava L2 Testnets. Please also have a look at the [Optimism Pectra update information](https://docs.optimism.io/notices/pectra-changes).
 
-See also the [Optimism Pectra update information](https://docs.optimism.io/notices/pectra-changes).
+The Pectra upgrade for Holesky L1 was activated on slot: 3710976 (Mon, Feb 24 at 21:55:12 UTC).
 
-:::info
-This page will be updated continuously with information and timelines. Ethereum L1 clients are currently scheduled to have a release by Feb 13th.
+:::danger
+The Pectra upgrade on Holesky caused a [chain split](https://x.com/TimBeiko/status/1895219970356949418) which in turn stalled the Celo Alfajores and Baklava networks.
```

Comment:
Alternative link, that also makes it clear it's ongoing:
https://github.com/ethereum/pm/blob/master/Pectra/holesky-postmortem.md

Reply:
Added some mroe content.